rust/release/src/main.rs

```diff
@@ -304,18 +304,26 @@ Overview:"#
                 println!("-------------------------\n");
                 println!("Branch to create {}", branch_name);
             } else {
-                // This pathing abuses the fact that we control where this is run from: CI
-                // and in CI this is *usually* run in the rust/release path
-                let ax_cargo = PathBuf::from("../actyx/ax/Cargo.toml").canonicalize()?;
-                let ax_core_cargo = PathBuf::from("../actyx/ax-core/Cargo.toml").canonicalize()?;
+                // We expect the binary to not have been moved outside of the target folder
+                let cwd = current_exe()?;
+                let actyx_folder = cwd
```

Comment:
If my counting is correct, then this should be named `rust_folder` instead.

Reply:
It is. Nice catch